src/index.js

```diff
@@ -29,9 +32,11 @@ import { isRemote } from 'pouchdb-utils';
 
             let oldDataFunc = this.$options.data;      
 
-            this.$options.data= function(vm) {
+            // the data function is explicitly passed a vm object in 
+            // we're calling it veem here to differentiate with vm which is used often in this plugin
+            this.$options.data= function(veem) {
```

Comment:
whats veem?

Reply:
veem was silly. I'll switch it back to vm. I was trying to differentiate between the two different vms that are in different scopes. No need